--- origsrc/darktable-0.9.2/cmake/modules/FindGettext.cmake	2011-08-24 04:20:29.000000000 -0500
+++ src/darktable-0.9.2/cmake/modules/FindGettext.cmake	2011-09-20 17:31:00.909117200 -0500
@@ -12,7 +12,7 @@ include(LibFindMacros)
 
 # On Linux there is no pkgconfig script, but with this we force Gettext_PKGCONF_INCLUDE_DIRS to ""
 libfind_pkg_check_modules(Gettext_PKGCONF Gettext)
-if(WIN32 OR APPLE)
+if(WIN32 OR APPLE OR CYGWIN)
   set(Gettext_LIBRARY_SEARCH_DIRS
     ${Gettext_PKGCONF_LIBRARY_DIRS}
     /opt/local/lib
